policy/datasets.yaml

```diff
@@ -589,6 +589,11 @@ srcMatchFull:
     python: lsst.afw.table.BaseCatalog
     tables: raw
     template: ''
+jointcal_config:
+    persistable: Config
+    python: lsst.pipe.tasks.processCcd.ProcessCcdConfig
```

Comment:
I believe this should be `lsst.jointcal.JointcalConfig`

Reply:
Good catch! I wouldn't have even thought to check that.

Wish we had a more general way to test that the configs are persisted correctly.

Reply:
I don't know if it's general, but as I commented on the ticket if you check the datatype of what the butler gives you back then I think that would do it in two lines.  A basic round-trip test.

Reply:
The butler already does that: my test in jointcal uses that and more.